packages/authorization/src/authorize-interceptor.ts

```diff
@@ -83,7 +78,7 @@ export class AuthorizationInterceptor implements Provider<Interceptor> {
     debug('Current user', user);
 
     const authorizationCtx: AuthorizationContext = {
-      principals: user ? [userToPrinciple(user)] : [],
+      principals: user ? [user] : [],
```

Comment:
@derdeka We still need a converter function to add `name` and `type` for `Principal` like:
```ts
// I am fixing the same issue and here is my local change. I am adding tests for it
// and that's why haven't pushed it to remote.
function userToPrinciple(user: UserProfile): Principal {
  return {
    ...user,
    name: user.name || user[securityId],
    type: 'USER',
  }
```

Can you keep the converter and add some test for it?